### DISCUSSION TITLE
Vercel Templates page breaks when '[' is typed in the search filter

### DISCUSSION THREAD

**Discussion by spaultify** [vercel/community#3434] [Bug Report]
### Page to Investigate

https://vercel.com/templates

### Steps to Reproduce

1. Go to https://vercel.com/templates
2. Find the search bar under "Filter Templates"
3. Type `[` (open square bracket)
4. The page should perform some erratic rendering, ultimately leading to a blank page.
5. Pressing the navigate back button (back to the landing page) and navigate forward button (re-navigate to the Templates page) will not solve the issue. However, navigating back to the landing page and clicking Templates will solve the issue
6. Typing other characters such as `?` will also break the Templates page. Inspecting the console shows that this issue is related to the "regex function" that triggers whenever the search input changes.

**Comment by swarnava** [comment]
Hi,

Thank you for reporting. I have forwarded this to our Engineering team to take a closer look. Have a good day!

**Comment by swarnava** [comment]
This should be fixed now.
